packages/fluentui/react-northstar/src/components/Breadcrumb/Breadcrumb.tsx

```diff
@@ -74,16 +74,16 @@ export const Breadcrumb = compose<'nav', BreadcrumbProps, BreadcrumbStylesProps,
     const unhandledProps = useUnhandledProps(composeOptions.handledProps, props);
     const ElementType = getElementType(props);
 
-    const result = getA11yProps.unstable_wrapWithFocusZone(
+    const result = (
       <ElementType
```

Comment:
@layershifter if we add the `unstable_wrapWithFocusZone` wrapping `<ElementType />` we get error from `isConformant` for not passing `ref` to root component. For now we moved to inside wrapping the list, is it proper?

Reply:
It can work there, but we need to fix it in `unstable_wrapWithFocusZone`

Reply:
Should I open a issue for that?

Reply:
pinged shift and got reply:

> I don't think so as it's unclear what we would like to fix